x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -488,6 +489,25 @@ class PackagePolicyService {
         pkgName: packagePolicy.package.name,
         pkgVersion: installedPackage?.version ?? '',
       });
+
+      const isInstalledVersionLessThanOrEqualToPolicyVersion = semverLte(
+        installedPackage?.version ?? '',
```

Comment:
In what case does an installed package not have a version?

Reply:
The version isn't nullable here, but the `installedPackage` value is. It's probably worth throwing an error here instead of trying to account for non-installed packages in this case though so good catch.

Reply:
Ah yeah misread it, but that sounds good as it wasn't immediately clear to me how the empty string would perform in the semver check